WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -558,6 +556,14 @@ import WordPressComAnalytics
             presentViewController(alertController, animated: true, completion: nil)
         }
     }
+    
+    private func sharePost(postID: NSManagedObjectID, fromView anchorView: UIView) {
+        if let post = self.postWithObjectID(postID) {
```

Comment:
Any particular reason why the change would be necessary other than preference of one vs the other?

`if let` is a pretty standard way of avoiding `nil` values.